cli/src/opts.rs

```diff
@@ -92,9 +94,12 @@ pub struct SharedArgs {
     ///
     /// The `guest` option can be additionally configured as:
     ///
-    ///     --profile=guest[,path]
+    ///     --profile=guest[[,path],sample]
```

Comment:
```suggestion
    ///     --profile=guest[,path[,sample]]
```

`[]` indicate a layer of optionality, yes?

Based on the parser further down, you can have `path` alone (good, keeps backwards compatibility), or `path` and `sample`, but not `sample` without `path`.

Reply:
Yeah, I definitely had this wrong.